### ISSUE TITLE
Branch status for libunifex

### ISSUE THREAD

**Issue by ccotter** [facebookexperimental/libunifex#435]
I noticed most of the recent PRs to libunifex have landed on broken-stdlib rather than main. Is there a description of which features or bug fixes land where? Is there a plan to eventually merge the broken-stdlib branch into main, or something else? Thanks!

**Comment by AnujYamdagni** [comment]
Hey @ccotter! Yes most of our recent commits have been in broken-stdlib but we’re currently formalizing a plan to merge unstable and broken-stdlib into main. Unfortunately there’s no ETA for the merge yet but stay tuned.

**Comment by ccotter** [comment]
@AnujYamdagni - i see the unstable branch recently merged into broken-stdlib. have you had a chance to move towards a plan on merging main and broken-stdlib?

By the way, if there's anything i can do to help, please let me know (although I can understand if this might be too involved for outside help). We've started using this library (based on the main branch) at my company, and I backported one algorithm (`when_all_range` + associated required utilities like unifex::optional) from the broken-stdlib branch in our local branch that's derived from the main branch here.

I'm generally interested in knowing the longer term plan for unifex (e.g., continued investment/support from Meta?) as we consider having other teams in my company look at onboarding this library. As other teams onboard things like coroutines, we'd like to have a solid foundation of a Sender/Receiver vocabulary and base algos, and unifex is a great candidate (one day we'd probably migrate to `std::execution`, if/when that is accepted into the standard library). If you'd be able to provide any color on that, please let me know!

**Comment by janondrusek** [comment]
@ccotter Apologies for a late reply. Historically there have been 3 branches:

1. `main` (highest bar for PR review)
2. `unstable` (lower PR bar, incubator for `main` + scheduler affinity #431 for `unifex::task`)
3. `broken-stdlib` (`absl::` polyfills for incomplete / incorrect `std::` implementations on some older devices, internally used at Meta)

I merged `unstable` (now deleted) into `broken-stdlib` and Meta does not need the polyfills anymore. Here is a strategy for the next couple of weeks:

1. tag current `main`, e.g. `1.0.0`
2. (in parallel) remove `absl::` polyfills from `broken-stdlib`
3. merge all the new algorithms from `broken-stdlib` into `main` and tag e.g. `1.1.0`
4. merge scheduler affinity into `main`
5. delete `broken-stdlib`

**Comment by janondrusek** [comment]
Both `broken-stdlib` and `unstable` are merged into `main` and deleted.
